controlers/users.js

```diff
@@ -70,10 +77,12 @@ const logIn = async (req, res, next) => {
         },
       });
     } else {
-      throw new Error('Wrong password');
+      throw new Error('Wrong password or not verified');
```

Comment:
Z logicznego punktu treść tego errora nic nie zmienia może być pusty. Chyba że w catch byś go odczytywał czy coś

Reply:
Masz całkowitą racje. Jednak error bez treści średnio pokazuje w którym miejscu się coś się wykrzacza. Dlatego wolę dać cokolwiek, co pokazuje że... ahaa.. to jest ten error jeśli coś się pisze i consol loguje. Druga rzecz to konkretny komunikat też daje opisowe info. Przy zadanym responsie bez szczegółów błędu to ten błąd się nie pokaże, ale jeśli błąd miałby być widoczny w responsie to już wiadomo coś więcej :)